porick/lib.py

```diff
@@ -15,11 +51,14 @@ def current_page(default=1):
         return default
 
 
+def hash_password(plaintext):
```

Comment:
Great. Can you also change https://github.com/stesh/porick-flask/blob/stephen/password_reset/porick/lib.py#L112 to call this function?